server/api/models/posts.js

```diff
@@ -0,0 +1,59 @@
+const db = require ('../dbConfig')
+
```

Comment:
I like the use of OOP here, makes the Routing cleaner and keeps logic separate and easier to read, again in line with MVC pattern.